### DISCUSSION TITLE
Is there a guide or reference how to create plugins and use them?

### DISCUSSION THREAD

**Discussion by MaksymOsovitnii** [hcengineering/platform#6032] [Q&A]
Hi all. Is there any guide or reference how to create plugins and use them in the platform. I have checked everything but havent found any instructions.

**Comment by stephanboersma** [comment]
Bump, I am also looking for this to review the possibility to extend with custom functionality :)

**Comment by wienans** [comment]
I assume there is no plugin system which is capaulated from the main source code?
